src/main/puzzle/scenario/scenario-settings.gd

```diff
@@ -43,6 +43,10 @@ var score := ScoreRules.new()
 # accomplishments such as surviving 10 minutes or getting 1,000 points. 
 var win_condition := Milestone.new()
 
+func _init() -> void:
+	set_start_level("0")
```

Comment:
This is worth documenting in a code comment and commit comment, it's a pretty unintuitive change.

Reply:
Added code comment, commit comment